### PR TITLE
onroad test fixes

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

File: selfdrive/test/test_onroad.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "./loggerd": 45.0,
   "./locationd": 9.1,
   "selfdrive.controls.plannerd": 22.6,
-  "./_ui": 15.0,
```
Comment (adeebshihadeh) [inline selfdrive/test/test_onroad.py:29]: @deanlee any idea why this went up after the recent refactors?

Comment (deanlee) [reply selfdrive/test/test_onroad.py:29]: imo, the new OnroadHud window Increased the workload of window compositing. 
we may need to consider using plugin(https://doc.qt.io/qt-5/embedded-linux.html) or QGraphicsScene(with opengl viewport) to reduce the cpu usage if necessary.

**Comment by deanlee** [comment]
@adeebshihadeh qt constantly re-upload the raster memory buffers (onroadHud & OnroadAlerts painted on raster memory) into OpenGL textures for compositing, this can have a significant impact on performance.

**Comment by deanlee** [comment]
A possible quick solution is to keep the existing structures and call hud and alerts's  paint functions in NvgWindow::paintGL

**Pull request merged**